packages/account-v2/build/build-loaders.ts

```diff
@@ -0,0 +1,115 @@
+import { RuleSetRule } from 'webpack';
+import { TBuildOptions } from './types/build-types';
+
+export const buildLoaders = (options: TBuildOptions): RuleSetRule[] => {
+    const svgLoaders = [
+        {
+            loader: 'babel-loader',
+            options: {
+                cacheDirectory: true,
+                rootMode: 'upward',
+            },
+        },
+        {
+            loader: 'react-svg-loader',
+            options: {
+                jsx: true,
+                svgo: {
+                    floatPrecision: 3,
+                    plugins: [
+                        { removeTitle: false },
+                        { removeUselessStrokeAndFill: false },
+                        { removeUnknownsAndDefaults: false },
+                        { removeViewBox: false },
+                    ],
+                },
+            },
+        },
+    ];
+    const babelLoader = {
+        exclude: /node_modules/,
+        test: /\.(js|jsx|ts|tsx)$/,
+        use: [
+            {
+                loader: 'babel-loader',
+                options: {
+                    cacheDirectory: true,
+                    rootMode: 'upward',
+                },
+            },
+        ],
+    };
+    const styleLoader = {
+        test: /\.(sc|sa|c)ss$/,
+        use: [
+            'style-loader',
+            {
+                loader: 'css-loader',
+                options: {
+                    modules: {
+                        auto: (path: string) => path.includes('.module.'),
+                        localIdentName: options.isRelease ? '[hash:base64]' : '[path][name]__[local]',
+                    },
+                    url: true,
+                },
+            },
+            {
+                loader: 'postcss-loader',
+                options: {
+                    postcssOptions: {
+                        config: options.paths.root,
+                    },
+                },
+            },
+            {
+                loader: 'resolve-url-loader',
+                options: {
+                    keepQuery: true,
+                    sourceMap: true,
+                },
+            },
+            // 'sass-loader',
+            // {
+            //     loader: 'sass-resources-loader',
+            //     options: {},
+            // },
```

Comment:
If this is not needed and it works without it I think we can just remove in my opinion.